### PR TITLE
fixed letter sizes

### PR DESCRIPTION
**Pull request by lawniczek-piotr**
webpack didn't resolved SimpleMDE and Marked plugins so changed to lowercase

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ module.exports = {
     umdNamedDefine: true
   },
   externals: {
-    'simplemde': 'SimpleMDE',
-    'marked': 'Marked'
+    'simplemde': 'simplemde',
```
Comment (F-loat) [inline webpack.config.js]: I think this line should be `'simplemde': 'SimpleMDE',`.

Comment (F-loat) [reply webpack.config.js]:
``` html
<script src="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.js"></script>
```
``` javascript
<script>
var simplemde = new SimpleMDE();
</script>
```

Comment (lawniczek-piotr) [reply webpack.config.js]: I tried that and npm warned that cannot resolve 'Marked' and 'SimpldeMDE'. Lower case version compiled without any errors

Comment (F-loat) [reply webpack.config.js]: emmmm...sorry, my mistake. we can use an object to describe all the possible ways. ([doc](https://webpack.js.org/configuration/externals/))
``` javascript
externals : {
  simplemde : {
    commonjs: 'simplemde',
    amd: 'simplemde',
    root: 'SimpleMDE' 
  },
  'marked': 'marked'
}
```

Comment (F-loat) [reply webpack.config.js]: If there is no problem, please update and bump package version to `0.4.6` .
Thanks.

**Pull request merged**